docs/en/pipeline-mgmt/basic.md

```diff
@@ -0,0 +1,26 @@
+# Basic configuration
+
+For the clickstream project, you can specify the basic configuration of the data pipeline and set the following configurations:
+
+* **AWS Region**: select the region where the pipeline is located. If you specify the region where AWS services are not available, the feature will be disabled by default. Check [the region table][region-table] for feature availability.
```

Comment:
**AWS Region**: select a region at which the pipeline will be created. If you select a region where some AWS services are not available, the corresponding feature will be disabled by default